test/parallel/parallel.status

```diff
@@ -5,6 +5,8 @@ prefix parallel
 # sample-test                        : PASS,FLAKY
 
 [true] # This section applies to all platforms
+# https://github.com/nodejs/node/issues/41123
+test-repl-sigint-nested-eval: PASS, FLAKY
```

Comment:
The test is skipped on Windows, but it is easier to do this than repeating the same stuff on all other supported platforms, so I have no objections.